index.rst

```diff
@@ -105,7 +109,8 @@ This section discusses how to derive the experimental measurement of the sensiti
 one starts from the matrix derived in section :ref:`optical_model`, which is purely diagonal. The general procedure
 of the test consists
 of aligning the system to the best of one's ability at a given elevation, then stepping the hexapod in small
-steps and measuring the WFE. Performing the test successfully requires a little more thought.
+steps and measuring the WFE. Fitting a line to this relationship gives the terms in the matrix.
+Performing the test successfully requires a little more thought and the following considerations should be made:
 
 1. Due to the numerous rotations in the system, it is best to choose a target that is transiting the meridian
```

Comment:
I think this itemized list is not being properly rendered. All the items are nested in a single paragraph. Can you take a look and fix it? Also, the phrase 

>  Choose a target near elevation 60.

seems a bit out of place. It is missing some context there.